anvill/src/Lifters/FunctionLifter.cpp

```diff
@@ -260,16 +290,106 @@ void FunctionLifter::VisitDirectJump(const remill::Instruction &inst,
   llvm::BranchInst::Create(GetOrCreateTargetBlock(inst.branch_taken_pc), block);
 }
 
-// Visit an indirect jump control-flow instruction. This may be register- or
-// memory-indirect, e.g. `jmp rax` or `jmp [rax]` on x86. Thus, the target is
-// not know a priori and our default mechanism for handling this is to perform
-// a tail-call to the `__remill_jump` function, whose role is to be a stand-in
-// something that enacts the effect of "transfer to target."
+// This is a list of the possibilities we want to cover:
```

Comment:
Update the comment to include part of the old comment. I'd like these comments to provide the kind of end-to-end context, i.e. this thing is related to `jmp rax`. Your current comment is good at describing the mechanics/corner cases.